patches/components-vector_icons-aggregate_vector_icons.py.patch

```diff
@@ -1,8 +1,16 @@
 diff --git a/components/vector_icons/aggregate_vector_icons.py b/components/vector_icons/aggregate_vector_icons.py
-index 3e5a94a7f053ffc66f1a820073265cbfe1332b39..0175ec714b441df5e8fa58e9391e2de6bbd67755 100644
+index 3e5a94a7f053f..2ccc03b6c819e 100644
 --- a/components/vector_icons/aggregate_vector_icons.py
 +++ b/components/vector_icons/aggregate_vector_icons.py
-@@ -102,7 +102,7 @@ def ExtractIconReps(icon_file_name):
+@@ -10,6 +10,7 @@ import re
+ import shlex
+ import sys
+ import textwrap
++from import_inline import inline_file_from_src; inline_file_from_src('brave/chromium_src/components/vector_icons/aggregate_vector_icons.py', globals(), locals())
```

Comment:
pls put this just before `def main():`. It should work the same, but this way it's possible to override everything declared in the file.